concepts/05 UI Components/Button/00 Getting Started with Button/08 Stylize the Button.md

```diff
@@ -0,0 +1,51 @@
+The Button supports several predefined color schemes. To specify one of them, set the [type](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#type) property. You can also use the [stylingMode](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#stylingMode) property to customize the fill and borders. Refer to the [Predefined Types](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Button/PredefinedTypes/) demo for more information about these properties.
```

Comment:
"To specify a color scheme" - sounds more like you create a new scheme, set it up. I'd say "apply" instead. 

```suggestion
The Button supports several predefined color schemes. To apply one of them, set the [type](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#type) property. You can also use the [stylingMode](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#stylingMode) property to customize the fill and borders. Refer to the [Predefined Types](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Button/PredefinedTypes/) demo for more information about these properties.
```